lib/Obfuscator.cpp

```diff
@@ -179,6 +179,7 @@ class ObfuscatorPluginAction : public PluginASTAction {
   }
 
   void EndSourceFileAction() override {
+    // Output to stdout
```

Comment:
Perhaps it's worth explaining that `llvm::outs()` is in practice `stdout`? For example: `Output to stdout (via llvm::outs()`.